nextflow_schema.json

```diff
@@ -366,16 +366,20 @@
                 },
                 "phix_reference": {
                     "type": "string",
-                    "default": "${baseDir}/assets/data/GCA_002596845.1_ASM259684v1_genomic.fna.gz",
                     "description": "Genome reference used to remove Illumina PhiX contaminant reads.",
-                    "hidden": true,
                     "format": "file-path",
-                    "exists": true
+                    "exists": true,
+                    "help_text": "If not supplied, and keeping phix is turned off, an included genome (GCA_002596845.1) will be used "
```

Comment:
I had linting problems with files in `assets` with defaults (discussed at Slack not so long ago). I chose to keep the default in `nextflow.config` but remove the `"default":` line from here and mention the default in the `"description":` instead.

Reply:
Yeah, see my comment above. We came up with another mechanism, I guess yours still works but I'm pretty sure linting will complain if there is a default in `nextflow.config` but not in `_schema.json` ? :thinking:

Reply:
Huh interesting... well anyway, if you don't mind I will leave our implementation as is and can revert it in the future?